redash/query_runner/big_query.py

```diff
@@ -230,7 +231,7 @@ def _get_project_id(self):
         return requests.get('http://metadata/computeMetadata/v1/project/project-id', headers={'Metadata-Flavor': 'Google'}).content
 
     def _get_bigquery_service(self):
-        credentials = gce.AppAssertionCredentials(scope='https://www.googleapis.com/auth/bigquery')
+        credentials = gce.AppAssertionCredentials(scope='https://www.googleapis.com/auth/bigquery,https://www.googleapis.com/auth/drive')
```

Comment:
I'm not sure this will work; I think the user needs to boot the instance with this API enabled.

If you didn't have the chance to test this with existing instance, let's just keep this fix limited to the "regular" BigQuery query runner.